### PR TITLE
Create welcoming home page layout

### PR DESCRIPTION
**Pull request by smiechuskce**
Website's welcoming "work in progress" home page layout.

## Description
Implemented the proposed layout of the home page. Should look good on smaller screens.

## Where should the reviewer start?
Build and run the solution.

## Motivation and context
#25 

## How has this been tested?
No tests. Just run the project and verified the design and look.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Program.cs [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
+                    var context = services.GetRequiredService<ApplicationDbContext>();
+                    DbInitializer.Seed(context);
+                }
+                catch (Exception ex)
+                {
+                    var logger = services.GetRequiredService<ILogger<Program>>();
+                    logger.LogError(ex, "An error occurred while seeding the database.");
+                }
+            }*/
+
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Program.cs]: remove commented code

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/css/site.css [hunk 2]
```diff
@@ -10,6 +9,73 @@
...
+
+.navbar-collapse.collapse, .navbar-collapse.collapsing {
+    font-size: 2em;
+    padding-top: 25px;
+    color: black;
+}
+
+.nav.navbar-nav {
+    display: flex;
+    justify-content: center;
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/css/site.css]: maybe better is right?

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -1,59 +1,10 @@
...
-    </script>
-}
-else
-{
-    <div class="row">
-        <div class="col-md-8">
-            <p>There is no open test right now - please wait for opening.</p>
-        </div>
+<div class="row">
+    <div class="col-md-8 content-centered">
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml]: Remove `col-md-8` - it looks better

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml [hunk 4]
```diff
@@ -1,59 +1,10 @@
...
+        <img id="santa" src="images/santa.JPG" alt="Święty Mikołaj" />
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml]: don't use diacritic signs

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/css/site.css (hunk 2 above)
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/css/site.css]: Changed to "flex-end" to be valid flex.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml [hunk 5]
```diff
@@ -18,32 +19,34 @@
...
+                    <img id="calendar-text" src="images/devadventcalendar.JPG" alt="DevAdventCalendar"/>
+                </a>
             </div>
             <div class="navbar-collapse collapse">
                 <ul class="nav navbar-nav">
-                    <li><a asp-area="" asp-controller="Home" asp-action="Index">Home</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="About">About</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="Contact">Contact</a></li>
+                    <li class="dropdown"><a asp-area="" asp-controller="Home" asp-action="About">O nas</a></li>
+                    <li class="dropdown"><a asp-area="" asp-controller="Home" asp-action="Contact">Kontakt</a></li>
```
[the remark this answers is not in the document]
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml]: Fixed.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml [hunk 6]
```diff
@@ -18,32 +19,34 @@
...
                 <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
-                    <span class="sr-only">Toggle navigation</span>
+                    <span class="sr-only">Otwórz menu</span>
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">DevAdventCalendarCompetition</a>
+                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">
+                    <img id="calendar-logo" src="images/logo.jpg" alt="Logo - prezent"/>
```
[the remark this answers is not in the document]
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml]: Fixed.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml [hunk 7]
```diff
@@ -18,32 +19,34 @@
...
             <div class="navbar-collapse collapse">
                 <ul class="nav navbar-nav">
-                    <li><a asp-area="" asp-controller="Home" asp-action="Index">Home</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="About">About</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="Contact">Contact</a></li>
+                    <li class="dropdown"><a asp-area="" asp-controller="Home" asp-action="About">O nas</a></li>
+                    <li class="dropdown"><a asp-area="" asp-controller="Home" asp-action="Contact">Kontakt</a></li>
                 </ul>
-                @await Html.PartialAsync("_LoginPartial");
+                @*@await Html.PartialAsync("_LoginPartial");*@
```
[the remark this answers is not in the document]
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml]: Fixed.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/About.cshtml [hunk 8]
```diff
@@ -4,19 +4,125 @@
...
-                <p>Główny organizator projektu od 2017 roku oraz programista. Bloguje na <a href="http://www.programmer-girl.com">www.programmer-girl.com</a> i aktywnie udziela się we wrocławskim świecie IT. Zaangażowana m. in. w organizację Wroc# Online.</p>
-            </div>            
+        <div class="about-person">
+            <div class="row">
+                <div class="col-md-2">
+                    <img src="~/images/photo_weronika_tobor.png" alt="Weronika Tobor" width="80" />
+                </div>
+                <div class="col-md-10">
+                    <h2>Weronika Tobor</h2>
+                    <p>Główny organizator projektu od 2017 roku oraz programista. Bloguje na <a href="http://www.programmer-girl.com">www.programmer-girl.com</a> i aktywnie udziela się we wrocławskim świecie IT. Zaangażowana m. in. w organizację Wroc# Online.</p>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/About.cshtml]: use everywhere in href `target="_blank"` (open in new tab)

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/About.cshtml [hunk 9]
```diff
@@ -4,19 +4,125 @@
...
+                </div>
+            </div>
+            <div class="row about-social-section">
+                <div class="col-md-4"><img src="~/images/icon_www.png" alt="WWW" /><a href="http://www.programmer-girl.com">www.programmer-girl.com</a></div>
+                <div class="col-md-4"><img src="~/images/icon_fb.png" alt="Facebook" />@@programmergirlblog</div>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/About.cshtml]: please add link _www.facebook.com/programmergirlblog_ (here and in other profiles)

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/About.cshtml [hunk 10]
```diff
@@ -4,19 +4,125 @@
...
+                <div class="col-md-4"><img src="~/images/icon_tt.png" alt="Twitter" />@@_programmergirl</div>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/About.cshtml]: please add link _www.twitter.com/_programmergirl_ (here and in other profiles)

**Comment by WTobor** [comment]
Contact page is not responsible (font should be resized automatically)
image

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Program.cs (hunk 1 above)
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Program.cs]: Done.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml [hunk 11]
```diff
@@ -18,32 +19,34 @@
...
-                    <span class="sr-only">Toggle navigation</span>
+                    <span class="sr-only">Otwórz menu</span>
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">DevAdventCalendarCompetition</a>
+                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">
+                    <img id="calendar-logo" src="images/logo.jpg" alt="Logo - prezent"/>
+                    <img id="calendar-text" src="images/devadventcalendar.JPG" alt="DevAdventCalendar"/>
```
[the remark this answers is not in the document]
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml]: Done. Set the minimal site width to 400 px.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml (hunk 3 above)
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml]: Replaced with col.

**Pull request merged**